socrata/socrata_upload.py

```diff
@@ -139,31 +161,34 @@ def upload(method, asset_id, sql_query, overwrite, year=None, township=None):
             + asset_id
         )
 
+    # We grab the data before uploading it so we can make sure timestamps are
+    # properly formatted
+    input_data = cursor.execute(sql_query, query_conditionals).as_pandas()
+    date_columns = input_data.select_dtypes(include="datetime").columns
+    input_data[date_columns] = input_data[date_columns].map(
+        lambda x: x.strftime("%Y-%m-%dT%X")
+    )
+    input_data = input_data.to_json(orient="records")
+
```

Comment:
Socrata expects are very particularly formatted timestamp.

Reply:
**[Question, non-blocking]** How does the Socrata agent handle this problem? Does it perform this conversion for us?

Continuing my thought above, this is another kind of thing that might be best solved by having dedicated views for our open data assets that exist on top of `default` views.